bin/embedding-sdk/release_utils.bash

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+function update_readme() {
```

Comment:
These functions are the same as when they were inlined in the SDK release workflow.